Quick Pad/MainPage.xaml.cs

```diff
@@ -100,8 +100,6 @@ public MainPage()
             AllFonts = new ObservableCollection<FontFamilyItem>(CanvasTextFormat.GetSystemFontFamilies().OrderBy(font => font).Select(font => new FontFamilyItem(font)));
             //
 
-            Clipboard.ContentChanged += Clipboard_ContentChanged;
```

Comment:
need this to call method to keep text in clipboard.

Reply:
I've added that back in a later commit and rename a method from Clipboard_ContentChanged to ClipboardStatusUpdate